backend/src/tests/test_crud.py

```diff
@@ -31,9 +31,10 @@ def patch_user_dao(self):
             with patch("app.auth.UserDAO", FakeUserDAO):
                 yield
 
-    def test_get_all(self, auth_client, fake_user):
-        make_todo(_id=1, title="todo1", content="foo", user=fake_user)
-        make_todo(_id=2, title="todo2", content="bar", user=fake_user)
+    @pytest.mark.asyncio
```

Comment:
I think the whole class can be marked @pytest.mark.asyncio, instead of needing to mark each method.

Reply:
neat.